proton-c/bindings/CMakeLists.txt

```diff
@@ -109,6 +109,11 @@ if (EMSCRIPTEN_FOUND)
   set (DEFAULT_JAVASCRIPT ON)
 endif (EMSCRIPTEN_FOUND)
 
+# Prerequisites for C++
```

Comment:
There is a foreach at the end of the file that sets up all the user-settable binding options. For each binding you must set a DEFAULT_FOO to be on or off depending if the requirements to build the binding are available, then the foreach sets up the BUILD_FOO option. CMAKE_CXX_COMPILER is set if a C++ compiler is available.